packages/mini-apps-ui-kit-react/src/components/BulletListItem/BulletListItem.tsx

```diff
@@ -0,0 +1,48 @@
+import { cn } from "@/lib/utils";
+import { ComponentProps, ReactNode, forwardRef } from "react";
+
+import { Typography } from "../Typography";
+
+const SparkIcon = (props: ComponentProps<"svg">) => (
```

Comment:
Looking at the design of some miniapps, this icon is only for demo. The idea is to use any icon for the disc

<img width="295" alt="image" src="https://github.com/user-attachments/assets/255a557e-21c1-4e4b-a852-4081b1243296" />

Reply:
@danielppolo I've refactored, please take a look. I tried to make it as much customizable as possible